views/editor.handlebars

```diff
@@ -285,7 +285,12 @@
 				email: '{{user.email}}'
 			})
 
-			mixpanel.identify('{{user._id}}')
+			dataLayer.push({
+				event: 'User info',
+				username: '{{user.username}}',
+				name: '{{user.name}}',
+				email: '{{user.email}}'
```

Comment:
do we need to send the email and name? would be nice not to play with this fire.